WordPress/src/main/res/layout/activity_log_list_footer_item.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We don't need the `FrameLayout` in this layout.  We can just use the `WPTextView`.

Reply:
I wasn't sure about that. Good point 👍